### ISSUE TITLE
🐛 Bug: Accessibility alt text bot action is failing

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [JoshuaKGoldberg/create-typescript-app#999]
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

The added action from #989 should be able to run.

### Actual

https://github.com/JoshuaKGoldberg/create-typescript-app/actions/runs/6697748677/workflow

```plaintext
Check failure on line 1 in .github/workflows/accessibility-alt-text-bot.yml

GitHub Actions / .github/workflows/accessibility-alt-text-bot.yml
Invalid workflow file
`issue` is not a valid event name
```

### Additional Info

cc @DerTimonius - maybe introduced during PR review? Is this something you've seen before?

**Comment by DerTimonius** [comment]
Hm.. I'm guessing the `if` is the problem. I will create a quick repo to test if it's really necessary and then fix the bug later today!

**Comment by DerTimonius** [comment]
The bug is a result of a typo... My bad...
